### DISCUSSION TITLE
Users management

### DISCUSSION THREAD

**Discussion by gruve-p** [btcpayserver/btcpayserver#3885] [Ideas / Feature Requests]
Currently we have an option to [sort users by email address](https://github.com/btcpayserver/btcpayserver/pull/2305) under server settings.

I would be great to have:
-  an option to sort them also by `Created`, `Verfied` and `Enabled` column
- a new checkbox colomn (before email) to select multiple users and perform bulk actions such as: `Resend verification email`, `Remove` `Enable` and `Disable` (instead doing them one by one, which is cumbersome if you have thousands of users).

**Comment by dstrukt** [comment]
We're going to be addressing the sort functionality across all table views, so will be sure to get this view updated when we get to that.

We're also addressing some (in)consistencies in the mass actions across all table views in the coming weeks as well, and having this functionality for the user tables makes sense, so this would fit in nicely.

Appreciate the feedback and clear signal this is a feature we should address!
